src/encoded/schemas/changelogs/file.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for file.json
 
+### Schema version 20
+* Reimplemented *run_type* dependency on fastq files.
```

Comment:
I would probably say "added upgrade required due to re-implementation of the run_type dependency